set.mm

```diff
@@ -712466,6 +712642,19 @@ Restricted quantification (extension)
     ( wb wral wi wa ralbiim ralbii r19.26 bitri ) ABGDFHZCEHABIDFHZBAIDFHZJZCEH
     PCEHQCEHJORCEABDFKLPQCEMN $.
 
+  $( Example for a theorem about a restricted universal quantification in which
+     the restricting class depends on (actually is) the bound variable:  All
+     sets containing themselves contain the universal class.  (Contributed by
+     AV, 24-Jun-2023.) $)
+  ralndv1 $p |- A. x e. x _V e. x $=
+    ( cvv cv wcel wel elirrv pm2.21i rgen ) BACZDZAIAAEJAFGH $.
+
+  $( Second example for a theorem about a restricted universal quantification
+     in which the restricting class depends on the bound variable: all subsets
+     of a set are sets.  (Contributed by AV, 24-Jun-2023.) $)
+  ralndv2 $p |- A. x e. ~P x x e. _V $=
+    ( cv cvv wcel cpw vex rgenw ) ABZCDAHEAFG $.
+
```

Comment:
Nice to have such examples at hand for reasoning about defining restricted quantification as ' A. x ( x e. A -> ph ) '